Documentation/admin/logging-destination.md

```diff
@@ -127,8 +127,117 @@ Installing the x-pack plugin on your Elasticsearch nodes enables authentication
 
 ### AWS Elasticsearch
 
-AWS Elasticsearch isn't officially supported at this time. AWS Elasticsearch functions differently from standard Elasticsearch making the standard Fluentd Elasticsearch plugin not work with it by default. If you are interested in using or contributing support for AWS Elasticsearch, please file an issue on [Github](https://github.com/coreos-inc/tectonic-installer/issues). Alternatively, you could look at one of the many AWS Signing proxies, which may work for your purposes, and allow you to use the default Elasticsearch configuration, pointed at the signing proxy.
+AWS Elasticsearch Service to quickly and easily run logging aggregation infrastructure with minimal overhead. Let's walk through the steps we're going to take to do this is to setup our Elasticsearch Service on AWS.
+  		  
+ - Log on to the AWS console and click Elasticsearch service.
+ - Click "Create a new domain".
+ - Configure your instance type, disk, and permissions. We'll be using IAM based credentials for security. If you're not familiar with IAM users, you can read more about it [here](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html). Here's an example config that I made specifically for this purpose. 
 
+If you do not wish to use credentials in your configuration via the `access_key_id` and `secret_access_key` options you should use IAM policies.
+
+The first step is to assign an IAM instance role `ROLE` to your EC2 instances. Name it appropriately. The role should contain no policy: we're using the possession of the role as the authenticating factor and placing the policy against the ES cluster.
+
+You should then configure a policy for the ES cluster policy thus, with appropriate substitutions for the capitalized terms:
+ 
+```
+{
+  "Version": "2017-08-14",
+   "Statement": [
+     {
+       "Effect": "Allow",
+       "Principal": {
+         "AWS": "arn:aws:iam::ACCOUNT:role/ROLE"
+       },
+       "Action": "es:*",
+       "Resource": "arn:aws:es:us-east-1:ACCOUNT:domain/ES_DOMAIN/*"
+     },
+     {
+       "Effect": "Allow",
+       "Principal": {
+         "AWS": "*"
+       },
+       "Action": "es:*",
+       "Resource": "arn:aws:es:us-east-1:ACCOUNT:domain/ES_DOMAIN/*",
+       "Condition": {
+         "IpAddress": {
+           "aws:SourceIp": "XXX.XX.XXX.XX/32"
+         }
+       }
+     }
+   ]
+}
+ 
+```
+This will allow your fluentd hosts (by virtue of the possession of the role) and any traffic coming from the specified IP addresses (you querying Kibana) to access the various endpoints. Whilst not ideally secure (both the fluentd and Kibana boxes should ideally be restricted to the verbs they require) it should allow you to get up and ingesting logs without anything getting in your way, before you tighten down the policy.
```

Comment:
This will allow fluentd hosts (by virtue of the possession of the role) and any traffic coming from the specified IP addresses (queries to Kibana) to access the listed endpoints. For greatest security, both the fluentd and Kibana boxes should be restricted to the verbs they require. This less secure example allows the cluster to begin ingesting logs before the policy is fully secured.

Reply:
Done!